packages/components/src/Notification/ToastNotification/subcomponents/ToastNotificationsList/ToastNotificationsList.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { HTMLAttributes } from "react"
+import { OverrideClassName } from "~types/OverrideClassName"
+import { GenericNotification } from "../../../subcomponents/GenericNotification"
```

Comment:
```suggestion
import { GenericNotification } from "~components/Notification/subcomponents/GenericNotification"
```

Reply:
Feels like not quite a subcomponent given it gets used like this, can reorg later